cypress/integration/pages/testsForAllCanonicalPages.js

```diff
@@ -16,6 +19,23 @@ export const testsThatFollowSmokeTestConfigForAllCanonicalPages = ({
 }) => {
   if (pageType !== 'errorPage404') {
     describe(`Running testsForAllCanonicalPages for ${service} ${pageType}`, () => {
+      describe('ATI', () => {
+        it('should have a noscript tag with an 1px image with the ati url', () => {
```

Comment:
This is going to run for ALLLL services, instead limit it to where there are differences.
I would write if smoke global var is true then run logic that's here, else run for only 1 PS, 1 WS and 1 GNL service.

Reply:
Shall I just manually choose the services that we test then, for example WS = Persian, PS = News, GNL = Japanese?